### DISCUSSION TITLE
[Feature] Add location via GPS coordinates

### DISCUSSION THREAD

**Discussion by Fmstrat** [immich-app/immich#11677] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently it's really hard to add something like a mountain overlook to the location via the UI. Search doesn't find the trailhead, and you'd need to go the exact coordinates anyway.

This feature request is to add the ability to "search" for standard coordinate formats like:
- `43.22247, -73.75896`
- `43°13'20.9"N 73°45'32.3"W`

And have the map go to that location:

![image](https://github.com/user-attachments/assets/55e316cb-cbd2-40a5-84f4-a05d08b0e48d)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I'm not currently at a pc to find the id, but this is definitely a duplicate of an existing request.

**Comment by Fmstrat** [comment]
@bo0tzz While there are other GPS requests, I couldn't find any that were specifically focused on coordinates. Apologies if I overlooked it, but it would be good to know which one.

**Comment by Fmstrat** [comment]
Just searched again to be sure, and no requests with "coordinate" or "coordinates" seems to align with this one.

**Comment by Fmstrat** [comment]
Nevermind! They were closed because of https://github.com/immich-app/immich/pull/11291

**Comment by Fmstrat** [comment]
@bo0tzz Though, after reviewing the merged PR, and the conversations going on there, could this be reopened if I edit it to focus on the "single field entry" that responders are requesting (since that PR is closed)?
